packages/react-native-otlp/README.md

```diff
@@ -0,0 +1,117 @@
+## React Native OTLP
+
+- Brief explanation TBD
```

Comment:
README looks good after this is filled in

Should note somewhere that after release we should also add this package to the list in the top level README.md file

Reply:
https://www.notion.so/embraceio/List-embrace-io-react-native-otlp-in-the-README-md-file-after-release-12a7e3c9985280c5b382c66633a1f946?pvs=4 created ticket for it